src/charts/utilization-bar/utilization-bar-chart.html

```diff
@@ -0,0 +1,34 @@
+<div class="pf-utilization-bar-chart">
+  <span ng-if="!layout || layout.type==='regular'">
+    <div ng-if="chartTitle" class="progress-description">{{chartTitle}}</div>
+    <div class="progress progress-label-top-right">
+      <div class="progress-bar" role="progressbar" ng-class="{'animate': animate,
+           'progress-bar-success': isOk, 'progress-bar-danger': isError, 'progress-bar-warning': isWarn}"
+           style="width:{{chartData.percentageUsed}}%" tooltip="{{chartData.percentageUsed}}% Used">
```

Comment:
Hmm, okay.  I don't have insight into that but to me it seems like as long as it looks the same it should be fine (since these are directives consumed by other projects).  By using ui-bootstrap instead of reinventing it we decrease the amount of code we have to maintain.